index.md

```diff
@@ -163,14 +163,16 @@ Border Not Represented             |  Border Is Represented
 :-------------------------:|:-------------------------:
 ![Border Not Represented](https://user-images.githubusercontent.com/19896216/144362121-37ca6e39-2698-4f48-95cb-16f3c3a31541.png)  |  ![Border Represented](https://user-images.githubusercontent.com/19896216/144362184-c0061707-abb6-443e-83f6-a9c8ac35f123.png)
 
-The only difference between the two representations is that one shows the border and the other does not. We found that this... [BLANK]
```

Comment:
Note for discussion tomorrow, do we want to run border representation for a2c/ppo?